packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -392,13 +390,26 @@ export class LitContracts {
       this.log('Your Provider(from signer):', this.provider);
     }
 
-    let addresses: any = await LitContracts.getContractAddresses(
+    const addresses: any = await LitContracts.getContractAddresses(
       this.network,
       this.customContext?.provider ?? this.provider,
       this.customContext
     );
 
-    this.log('resolved contract addresses for: ', this.network, addresses);
+    const logAddresses = Object.entries(addresses).reduce(
+      (output, [key, val]) => {
+        // @ts-expect-error since the object hash returned by `getContractAddresses` is `any`, we have no types here
+        output[key] = val.address;
+        return output;
+      },
+      {}
+    );
+
+    console.log(
```

Comment:
can we use `this.log` over `console.log`

Reply:
All set! Rebased to eliminate conflicts also.